woocommerce/woocommerce
# Remove legacy coupon note

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #34731

This PR removes CouponPageMoved class. I couldn’t find a way to forcibly display the legacy menu. New JN sites don’t display the legacy menu, and [display_legacy_menu](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/src/Internal/Admin/CouponsMovedTrait.php#L101) is not used anywhere currently. I think removing the note should be sufficient.




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

Create a new JN site and confirm the notes load without an error on `WooCommerce -> Home`




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Removed CouponPageMoved class.

</details>

<details>


<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
